pkgs/test/nixpkgs-check-by-name/src/main.rs

```diff
@@ -103,15 +93,40 @@ pub fn check_nixpkgs<W: io::Write>(
             }
             Ok(false)
         }
-        Success(_) => Ok(true),
+        Success(()) => Ok(true),
     }
 }
 
+/// Checks whether the pkgs/by-name structure in Nixpkgs is valid,
+/// and returns to which degree it's valid for checks with increased strictness.
```

Comment:
I'm not sure what this sentence is trying to communicate.

Reply:
Now updated, hope this helps: https://github.com/NixOS/nixpkgs/pull/272395/commits/413dd9c03e37562c61cd89799e5eb8a88c7bb42a#diff-fc467c6e2889ff0305612175c09cd06401123d3d5e8b6cfd1e85bec22586197eL102-R104